admin/partials/nginx-helper-general-options.php

```diff
@@ -393,9 +413,12 @@
 										echo wp_kses(
 											sprintf(
 												'%1$s<strong>%2$s</strong>%3$s<strong>%4$s</strong>.',
-												esc_html__( 'when a ', 'nginx-helper' ), esc_html__( 'comment', 'nginx-helper' ), esc_html__( ' is ', 'nginx-helper' ), esc_html__( 'unapproved/deleted', 'nginx-helper' )
+												esc_html__( 'when a ', 'nginx-helper' ),
+												esc_html__( 'comment', 'nginx-helper' ),
+												esc_html__( ' is ', 'nginx-helper' ),
+												esc_html__( 'unapproved/deleted', 'nginx-helper' )
```

Comment:
This is not correct way to make small string translatable. We need to allow entire string translatable.